contacts-csv-upload.mdx

```diff
@@ -29,15 +29,16 @@ Your CSV file should contain the following columns:
 - `contactAccountDomain` - Associated company's domain (e.g., "example.com")
 - `contactSourceId` - External identifier for the contact
 - `contractValue` - Associated contract value (numeric)
+- **`account_` prefix** - Columns with `account_` prefix will be stored as metadata on the associated account
```

Comment:
From a user perspective, it would be either `account_{X}` or `Account {X}`, right? 

Based on https://github.com/monterey-ai/monterey-ai-monorepo/blob/main/apps/worker/src/workers/ingestion/flatfile/utils.ts#L10

It will essentially turn a key like "Account Foo" into "account_foo" and also match.